website/docs/layouts/flex/partials/code/component-api.md

```diff
@@ -0,0 +1,78 @@
+## Component API
+
+### Layout::Flex
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="tag" @type="HTMLElementTagName" @default="div">
+    A valid HTML tag name to be used to render the flexbox element.
+  </C.Property>
+  <C.Property @name="direction" @type="enum" @values={{array "row" "column"}} @default="row">
+    The value of the CSS `flex-direction` property, which sets how the flex items (children) are placed in the flex container, defining the [main axis](https://developer.mozilla.org/en-US/docs/Glossary/Main_Axis) and the direction (for a technical explanation: [see MDN reference](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-direction)).
+    <br/><br/>
+    <em>Notice: we don't expose the "reverse" directions because they come with intrinsic accessibility limitations that we prefer our consumers to avoid.</em>
+  </C.Property>
+  <C.Property @name="justify" @type="enum" @values={{array "start" "center" "end" "space-between" "space-around" "space-evenly"}}>
+    The value of the CSS `justify-content` property, which defines how the space is distributed between and around content items along the [main axis](https://developer.mozilla.org/en-US/docs/Glossary/Main_Axis) of the flex container (for a technical explanation: [see MDN reference](https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content)).
+    <br/><br/>
+    <em>Notice: we expose only a subset of the values allowed for this property, covering only the most common use cases.</em>
+    <br/><br/>
+    <em>Tip: when the `@direction` is `row` this argument controls the horizontal alignment; when it's `column` it controls the vertical alignment.</em>
```

Comment:
Suggestion: it might be useful to move this up to the examples that use justify/align.